kernel/api/src/main/java/org/sakaiproject/email/api/AttachmentSizeException.java

```diff
@@ -0,0 +1,40 @@
+/**********************************************************************************
+ * Copyright 2008 Sakai Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **********************************************************************************/
+package org.sakaiproject.email.api;
+
+public class AttachmentSizeException extends Exception {
```

Comment:
Should the Exception reference the size some how?

Reply:
Added. New screenshot added to the jira.